aspnetcore/grpc/configuration.md

```diff
@@ -69,6 +69,10 @@ The following code:
 
 [!code-csharp[](~/grpc/configuration/sample/Program.cs?name=snippet&highlight=3-8)]
 
+Note that client interceptors aren't configured using `GrpcChannelOptions`. Instead, client interceptors are configured using the `Intercept` extension method with a channel. This extension method is in the `Grpc.Core.Interceptors` namespace.
```

Comment:
"using" in both sentences doesn't read well to me.

OK, or better way to word this?